src/module/rules/language.mjs

```
@@ -1,39 +1,39 @@
-    delete CONFIG.DND5E.languages.exotic
-    delete CONFIG.DND5E.languages.druidic;
-    delete CONFIG.DND5E.languages.cant;
+export function language() {
+    console.log("Elkan 5e  |  Initializing Languages");
+    if (!CONFIG.DND5E?.languages?.standard?.children) {
+        console.warn("Elkan 5e | CONFIG.DND5E.languages not properly initialized");
+        return
+    }
+    // Deleting Languages
+    const languagesToDelete = ["giant", "gnomish", "orc", "druidic", "cant"];
+    languagesToDelete.forEach(lang => CONFIG.DND5E.languages.standard.children[lang] = undefined);
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)